app/assets/stylesheets/content/_select2.scss

```diff
@@ -126,3 +115,75 @@
     margin-left: 0;
   }
 }
+
+//@todo for select2 visibility in livingStyleGuide, need another solution  it will rewrite globally
+.livingstyleguide--example {
+  overflow: visible !important;
+}
+
+//$se2- select2 var prefix
+$se2-theme-selectable-color: #f8f8f8;
+//todo change to global theme color var
+//{bgThemeHighlight}
+$se2-theme-selectable-color-highlighted: #24b3e7;
+
+$se2-element-height: 33px;
+$se2-line-height: 30px;
+$se2-button-width: 28px;
+$se2-arrow-button-width: 18px;
+$se2-results-max-height: 500px;
+
+$se2-max-width: 350px;
+$se2-width: 100%;
+
+.select2-container {
+  //Indirectly styling size of result list
+  position: relative;
+  max-width: $se2-max-width;
+  width: $se2-width;
+
+  .select2-choice {
+    border-radius: 0;
+    height: $se2-element-height;
+    line-height: $se2-element-height;
+    box-shadow: none;
+
+    > span {
+      line-height: $se2-line-height;
+    }
+    > div {
+      width: $se2-button-width;
```

Comment:
Properties should be ordered background-color, border-radius, line-height, text-align, width